packages/config/src/log/messages.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
```

Comment:
:+1: on starting to regroup all log messages from `@netlify/config` in the same file.